WorkloadManagementSystem/PilotAgent/PilotLogger/PilotLogger.py

```diff
@@ -0,0 +1,231 @@
+""" Pilot logger module for the remote loggin system
+    Only functions send and connect are depended on the client library used.
+    The current implementation uses stomp.
+"""
+
+__RCSID__ = "$Id$"
+
+import stomp
+import sys
+import Queue
+#Since PilotLogger can be used without DIRAC relaying only on PilotLoggerTools, we
+#check two possiblities:
```

Comment:
Since this is used only by the pilot, and this file as well as all the other files used by the pilot needs to be local, then we know the second option (from PilotLoggerTools import) will never fail. The first option thus becomes useless.